### DISCUSSION TITLE
bug with portals camera view being connected with player camera view

### DISCUSSION THREAD

**Discussion by Albinkizi** [iPortalTeam/ImmersivePortalsMod#1614] [Non-Compatibility Issue Report]
### Minecraft version

1.21.1

### Describe the issue

Testing version 6.0.1

Stuff that changes player camera view also changes portal camera view
**________________________________________________________________________________________________________________________________________**
**Problem 1**
When i have view bobbing on the portal also has view bobbing creating a wierd bug that makes the otherside of the portal wiggle so you can see through the blocks
![2024-09-01_12 34 06](https://github.com/user-attachments/assets/d7aaa5c7-565f-41c2-8c8d-1fafa38f42e5)

**Problem 2**
When player is hurt the screen tilts and the portal follows the tilt but in reverse as if the portal got hurt
![2024-09-01_12 50 56](https://github.com/user-attachments/assets/2bd2b4d0-8000-4816-b9ae-789a58983fbd)

**Problem 3**
When a potion effect renders on player the portals point view renders also the effect
**effects tested** 
Nausea
![2024-09-01_12 54 38](https://github.com/user-attachments/assets/b89dc483-b9e9-4b1c-b485-7374364529dc)

Minecraft netherportal effect "dont know why i tested it"
![2024-09-01_13 05 36](https://github.com/user-attachments/assets/5f69ea78-140e-47bb-9232-74d44d78c345)

**________________________________________________________________________________________________________________________________________**
fabric-0.16.3
fabric-api-0.103.0+1.21.1
immersive-portals-6.0.1-mc1.21.1-fabric

**Comment by qouteall** [comment]
Should be fixed in 6.0.2 https://modrinth.com/mod/immersiveportals/version/v6.0.2-mc1.21.1

**Reply by Albinkizi** [reply]
the ones in there is fix but f5 i forgot and is also broken
![2024-09-05_18 31 23](https://github.com/user-attachments/assets/a0b4c4a3-60c2-4492-841f-916ca9356157)

**Reply by qouteall** [reply]
@Albinkizi Fixed in 6.0.3
